lib/utils.js

```diff
@@ -95,10 +96,33 @@ function findClosestGitRepo(filePath) {
   return findClosestGitRepo(path.dirname(filePath));
 }
 
+// Returns a new Promise that resolve if the passed-in promise is rejected and
+// will be rejected with the provided error if the passed-in promise resolve.
+function reversePromise (promise, rejectionMessage) {
+  return new Promise((resolve, reject) => {
+    promise.then(() => reject(rejectionMessage)).catch(() => resolve())
+  })
+}
+
+// Spawns a child process and returns a promise that will be resolved if
+// the process ends with a code of 0, otherwise the promise will be rejected
+// with an error object of the provided rejectionType.
+// Optionally a callback
+function spawnPromise (cmd, cmdOptions, rejectionType) {
+  return new Promise((resolve, reject) => {
+    const proc = child_process.spawn(cmd, cmdOptions)
+    proc.on('close', (code) =>{
+      code ? reject({ type: rejectionType, data: proc.stderr }) : resolve()
```

Comment:
it may be worth resolving with `proc.stdout` so the user knows what the output was. wouldn't be used currently, but just a thought